ml/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOModel.scala

```diff
@@ -64,7 +64,8 @@ class H2OMOJOModel(val mojoData: Array[Byte], override val uid: String)
 
   def predictionSchema(): Seq[StructField] = {
     val fields = getOrCreateEasyModelWrapper().getModelCategory match {
-      case ModelCategory.Binomial => StructField("p0", DoubleType) :: StructField("p1", DoubleType) :: Nil
+      case ModelCategory.Binomial => StructField("p0", DoubleType) :: StructField("p1", DoubleType) ::
```

Comment:
Can ```p0```, ```p1``` and other columns contain ```null``` values?

Reply:
They can, do you think that is better solution than `nan`s? I was thinking about it but no strong opinion. `null` seemed to me a bit low-level, but otherwise no preference

Reply:
I'm asking because you don't specify nullability  in the constructor of ```StructField```. The default value is ```true```.  Spark optimizes generated whole-stage code if nullability is set to ```false``` (removes statements like ```if(value == null) ...``` from the generated code). It's just a nit anyway :-)

Reply:
Cool, didn't know that, thanks!